_includes/scripts.html

```diff
@@ -6,6 +6,17 @@
     <script src="{{ site.baseurl }}/js/responsive-tables.js"></script>
     <script>
         $(document).foundation();
+        try {
+            var paths = document.location.pathname.split('/').filter(
+                function(el) { return el.trim() !== ''});
+            var path = paths[paths.length-1];
+            $(".menu-subnav li a").each(function(i, el) {
```

Comment:
@waxenegger can we just add "header-subnav" here somehow to make this work for the omero-features tabs menu too?